src/v3/src/transformer/button/transformLaunchAuthenticatorButton.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { IDX_STEP } from '../../constants';
+import {
+  ButtonElement, ButtonType, DividerElement, TransformStepFnWithOptions,
+} from '../../types';
+import { loc } from '../../util';
+
+export const transformLaunchAuthenticatorButton: TransformStepFnWithOptions = ({
+  transaction,
+}) => (
+  formBag,
+) => {
+  const { neededToProceed: remediations } = transaction;
+  const containsLaunchAuthenticator = remediations.some(
+    (remediation) => remediation.name === IDX_STEP.LAUNCH_AUTHENTICATOR,
+  );
+  if (!containsLaunchAuthenticator) {
+    return formBag;
+  }
+
+  const launchAuthenticatorButton: ButtonElement = {
+    type: 'Button',
+    label: loc('oktaVerify.button', 'login'),
```

Comment:
I'm considering creating a new button just for this button since it's also used in the other two `launch-authenticator` views with varying behaviors. The impetus here is that the `Button` will need the Icon but no loading state, and the existing button is heavily laden with logic already, and I would have to decide how the button should behave if `loading=true` AND there is an Icon set. I think separating out into a new component (`LaunchAuthenticatorButton` perhaps) would sidestep this issue and also simplify the component.

Any thoughts?